app/services/event_service.rb

```diff
@@ -9,21 +9,21 @@ class << self
     include ActionView::Helpers::UrlHelper
   end
 
-  sig { params(work: Work, user: User, description: String).returns(T.nilable(Integer)) }
+  sig { params(work: Work, user: User, description: String).returns(T.untyped) }
```

Comment:
The return value is either an integer or an array of procs, depending on what the ActionCable broadcast does. We don't care about the return value, though, so let Sorbet ignore it.